### DISCUSSION TITLE
What happens with the temporary scratch file in case of interrupt virtnbdbackup?

### DISCUSSION THREAD

**Discussion by slu-010101** [abbbi/virtnbdbackup#201] [General]
What happens with the scratch file in case of interrupt the backup (power loss, reboot,...)?
Is this file not part of the VM disk which will merge after the backup is done?

**Comment by abbbi** [comment]
the scratch file is entirely handled by the backup process from libvirt side, i dont know in detail how libvirt handles these on powerloss. During reboot i assume the libvirt daemon should handle these situations gracefully.
